FWCore/Framework/interface/EventSetupRecord.h

```diff
@@ -109,6 +111,10 @@ namespace edm {
 
       template <typename HolderT>
       bool get(char const* iName, HolderT& iHolder) const {
+        if
+          UNLIKELY(requireTokens_) {
```

Comment:
`clang-format` does not like `UNLIKELY`?

Reply:
@makortel I think we already saw this at the time of the mass migration